tests/test_changesets.py

```diff
@@ -53,9 +55,35 @@ def test_cast():
     assert changeset.changes == {"name": "Richard"}
 
 
-def test_cast_type_check():
-    changeset = Changeset(User()).cast({"name": 1}, ["name"])
-    assert changeset.errors == {"name": ["Not a valid string."]}
+def test_cast_unknown_type():
+    changeset = Changeset(User()).cast({"unknown": 1}, ["unknown"])
+    assert changeset.changes == {}
+
+
+@pytest.mark.parametrize(
+    "key,val,valid,error",
```

Comment:
Could probably just have `key,val,error` here and just check if `if error` instead of checking `if valid`